query.go

```diff
@@ -435,37 +444,16 @@ func (q *query) allowClosestRejectedPeer() {
 
 	if q.dFilter != nil {
 		// If the closest non-diverse/rejected peer is closer to the key than the furthest
-		// peer among the Beta closest and queried peers:
-		// 1. If the rejected peer hasn't been queried, simply change it's state to "heard" and whitelist it in the filter so it can be queried.
-		// 2. If the rejected peer has already been queried, change it's state to "queried" and queue up all results returned by it for querying
-		//     by changing their state to "heard".
-
+		// peer among the Beta closest and queried peers simply change it's state to "heard" and whitelist it in the filter so it can be queried.
```

Comment:
```suggestion
		// peer among the Beta closest and queried peers, simply change it's state to "heard" and whitelist it in the filter so it can be queried.
```

Reply:
Done.